modules/ocf/manifests/packages/firefox.pp

```diff
@@ -3,14 +3,20 @@
     stage => first,
   }
 
-  $firefox_pkg = $::lsbdistcodename ? {
-    'jessie' => 'firefox',
-    default  => 'firefox-esr',
+  if $::lsbdistcodename == 'jessie' {
```

Comment:
No, I think this is fine for now, but I'll keep looking at `apt-show-versions`, maybe it would be worth using.